library/build.gradle

```diff
@@ -1,5 +1,12 @@
 apply plugin: 'com.android.library'
 
+ext {
+    versions = [
+            hamcrest: '1.3',
+            mockito : '2.10.0',
```

Comment:
Since this is a big version change, would it be a good idea to get onto the latest versions of these? 2.18.5 for mockito, for example?